src/solvers/CMakeLists.txt

```diff
@@ -122,6 +125,9 @@ elseif("${sat_impl}" STREQUAL "cadical")
 
     add_library(cadical STATIC IMPORTED)
 
+    if(ipo_supported)
+      set_property(TARGET cadical PROPERTY INTERPROCEDURAL_OPTIMIZATION True)
+    endif()
```

Comment:
How effective is this going to be? I am asking for `-march=native` will (AFAIK) not be forwarded to the cadical build, but perhaps we need to find a way of doing so?

Reply:
I thought that subprojects inherit the cache, so it would inherit -march=native and ipo_supported. I'll check, this was years ago

Reply:
The cadical build is done at configure time (https://github.com/diffblue/cbmc/blob/develop/src/solvers/CMakeLists.txt#L117), I don't think this is a subproject? (Unlike the minisat/glucose builds.)